java/src/jmri/jmrit/beantable/LRouteTableBundle_ca.properties

```diff
@@ -25,7 +25,7 @@ InputTitle = Selecciona l'estat del Disparador d'entrada per a aquesta L'Itinera
 MiscTab = Alineaci\u00f3 / Blocatge / etc.
 MiscTabHint = Selecciona script i fitxer de so, sensors d'alineaci\u00f3, altres par\u00e0metes
 MiscTitle = Accions L'Itinerari adicionls
-LRouteType = Tipus de L\u00f2gica d'Itinerari
+L'ItinerariType = Tipus de L\u00f2gica d'Itinerari
```

Comment:
I done a replace and I don't view that there is code in the grey part.


Sorry


El 19/08/2016 a les 18:37, Egbert Broerse ha escrit:
>
> In java/src/jmri/jmrit/beantable/LRouteTableBundle_ca.properties 
> <https://github.com/JMRI/JMRI/pull/1804#discussion_r75513012>:
>
> > @@ -25,7 +25,7 @@ InputTitle = Selecciona l'estat del Disparador d'entrada per a aquesta L'Itinera
> >  MiscTab = Alineaci\u00f3 / Blocatge / etc.
> >  MiscTabHint = Selecciona script i fitxer de so, sensors d'alineaci\u00f3, altres par\u00e0metes
> >  MiscTitle = Accions L'Itinerari adicionls
> > -LRouteType = Tipus de L\u00f2gica d'Itinerari
> > +L'ItinerariType = Tipus de L\u00f2gica d'Itinerari
>
> Also, never translate the word before the =
> When you do, the whole line is not found by the program as it looks 
> for that same unstranslated word in all the .properties files.
> So this should stay |LRouteType =|
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/JMRI/JMRI/pull/1804/files/c818482d84a6687ea1cfe5a77973dc0441b298c1..ac6fcec821702a36f2659b3691c95350e4ee97f1#r75513012>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AUET1fGAzDhNVClp9M22crBTa8vPR7xJks5qhdvbgaJpZM4JlwXK>.
>